rpc/legacyrpc/methods.go

```diff
@@ -1921,7 +1921,8 @@ func listTransactions(s *Server, icmd interface{}) (interface{}, error) {
 		// For now, don't bother trying to continue if the user
 		// specified an account, since this can't be (easily or
 		// efficiently) calculated.
-		return nil, errors.E("transactions can not be searched by account")
+		return nil, errors.E("Transactions can not be searched by account." +
+			" Use \"*\" to reference all accounts.")
```

Comment:
Instead of escaping, you can use a literal string with backticks.  Hard to provide an example here because it's mangled with the markdown formatting.

Reply:
https://play.golang.org/p/8jCytNXA3co

Reply:
The problem with backticks is that it would pass 80 columns because it would read the line break otherwise. https://play.golang.org/p/LVFfzW-agbn

Is it a problem?